src/library/tests/test_views.py

```diff
@@ -87,36 +131,37 @@ def test_quest_list_view(self):
         self.assertNotEqual(len(response.context['library_quests']), non_library_quest_count)
 
     def test_import_non_existing_quest_to_current_deck(self):
```

Comment:
rename to `test_import_quest_to_current_deck__non_existing_quest()`

The view method being tested is: `import_quest_to_current_deck()`

rename the tests to match our naming convention? It should be test_[view_method_name]__[what_is_being_tested_on_this_view]()